overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -51,7 +51,7 @@ func (f *fakeStore) SnapAction(_ context.Context, currentSnaps []*store.CurrentS
 	if len(actions) == 1 && actions[0].Action == "install" {
 		return []*snap.Info{{
 			SideInfo: snap.SideInfo{
-				RealName: actions[0].Name,
+				RealName: actions[0].InstanceName,
```

Comment:
here we should probably panic for now if InstanceName has a key part, otherwise this code will do incorrect things. Don't know if we have similar problems in other tests changes

Reply:
Added a panic